src/GeneratorV2/Process/Processor.cs

```diff
@@ -335,7 +335,8 @@ public static BaseCSType MakeCSType(GLType type, string? group, out Expression?
                 case GLBaseType bt:
                     return bt.Type switch
                     {
-                        PrimitiveType.Void => new CSVoid(),
+                        PrimitiveType.Void => new CSVoid(bt.Constant),
+                        PrimitiveType.Bool => new CSType("bool", bt.Constant),
```

Comment:
Are we sure this is getting marshalled correctly? Is `GLboolean` a byte or an int?

Reply:
its a byte. Im not sure what you mean by marshalled correctly

Reply:
In some flavors of C booleans are 4 bytes big. I'm just asking if the size is correct.